test/unit/dom/appendChild.js

```diff
@@ -221,4 +221,31 @@ describe('dom: appendChild', () => {
   runTests('slot');
   runTests('host');
   runTests('root');
+
+  describe('fragment:', () => {
```

Comment:
That was my original approach, and the vast majority of the tests failed, so I switched to breaking it out to be specific to fragment.

Fragment is fundamentally different from the others, so I was comfortable with this deviation.